torch-sys build error: .include(..) vs .includes(..)

I am updating a project I wrote a while ago to the use latest tch-rs and am freshening up all of its few dependencies.

Although on my machine I can clone the github repo and run the mnist example just fine, I am encountering a torch-sys build error in this project. I am careful to cargo clean before compiling my updated project.

The error is:

```
error[E0599]: no method named `includes` found for mutable reference `&mut Build` in the current scope
   --> /home/username/.cargo/registry/src/index.crates.io-6f17d22bba15001f/torch-sys-0.13.0/build.rs:373:22
    |
369 | /                 cc::Build::new()
370 | |                     .cpp(true)
371 | |                     .pic(true)
372 | |                     .warnings(false)
373 | |                     .includes(&self.libtorch_include_dirs)
    | |                     -^^^^^^^^ help: there is a method with a similar name: `include`
    | |_____________________|
    | 

error[E0599]: no method named `includes` found for mutable reference `&mut Build` in the current scope
   --> /home/username/.cargo/registry/src/index.crates.io-6f17d22bba15001f/torch-sys-0.13.0/build.rs:388:22
    |
384 | /                 cc::Build::new()
385 | |                     .cpp(true)
386 | |                     .pic(true)
387 | |                     .warnings(false)
388 | |                     .includes(&self.libtorch_include_dirs)
    | |                     -^^^^^^^^ help: there is a method with a similar name: `include`
    | |_____________________|
```

I have tried using both:

```
"tch" = { git = "https://github.com/LaurentMazare/tch-rs" }
```
and
```
tch = "0.13.0"
```

--- a/torch-sys/Cargo.toml
+++ b/torch-sys/Cargo.toml
@@ -17,7 +17,7 @@ libc = "0.2.0"
 
 [build-dependencies]
 anyhow = "1.0"
-cc = "1.0"
+cc = "1.0.61"
 ureq = { version = "2.6", optional = true, features = ["json"] }
 serde_json = { version = "1.0", optional = true }
 serde = { version = "1.0", optional = true, features = ["derive"] }

